content/tagging/assigning_tags.md

```diff
@@ -13,25 +13,195 @@ further_reading:
+```
+
+or
+
+```
+tags:
+    - key_first_tag:value_1
+    - key_second_tag:value_2
+    - key_third_tag:value_3
+```
+
+It is recommended you assign tags as `key:value` pairs but simple tags are also accepted. See [how to use][61] tagging for more details.
+
+### Environment Variables
+
+When installing the containerized Datadog Agent, host tags can be set using the environment variable `DD_TAGS`. We automatically collect common tags from [Docker][77], [Kubernetes][78], [ECS][79], [Swarm, Mesos, Nomad and Rancher][77]. To extract even more tags, use the following options:
+
+| Environment Variable               | Description                                    |
+|------------------------------------|------------------------------------------------|
+| `DD_DOCKER_LABELS_AS_TAGS`         | extract docker container labels                |
+| `DD_DOCKER_ENV_AS_TAGS`            | extract docker container environment variables |
+| `DD_KUBERNETES_POD_LABELS_AS_TAGS` | extract pod labels                             |
+
+**Examples:**
+
+```shell
+DD_KUBERNETES_POD_LABELS_AS_TAGS='{"app":"kube_app","release":"helm_release"}'
+DD_DOCKER_LABELS_AS_TAGS='{"com.docker.compose.service":"service_name"}'
+```
+
+Either define the variables in your custom `datadog.yaml`, or set them as JSON maps in these envvars. The map key is the source (`label/envvar`) name, and the map value is the Datadog tag name.
+
+### UI
+
+{{< tabs >}}
+{{% tab "Infrastructure List" %}}
+
+You can assign host tags in the UI via the [Infrastructure List][62]. Click on any host to show the host overlay on the right. Then under the *User* section, click the **Edit Tags** button. Enter the tags as a comma separated list then click **Save Tags**.
+
+{{< img src="tagging/assigning_tags/hostuitags.png" alt="Infrastructure List Tags" responsive="true" style="width:80%;">}}
+
+[62]: /graphing/infrastructure/
+
+{{% /tab %}}
+{{% tab "Monitors" %}}
+
+From the [Manage Monitors][63] page, select the checkbox next to each monitor to add tags (you can select one or multiple monitors). Click the **Edit Tags** button. Enter a tag or select one used previously. Then click *Add Tag `tag:name`* or *Apply Changes*. If tags were added previously, you can assign multiples at once using the tag checkboxes.
+
+{{< img src="tagging/assigning_tags/monitortags.png" alt="Manage Monitors Tags" responsive="true" style="width:80%;">}}
+
+When creating a monitor, assign monitor tags under step 4 *Say what's happening*:
+
+{{< img src="tagging/assigning_tags/monitorindivdualtags.png" alt="Create Monitor Tags" responsive="true" style="width:80%;">}}
+
+[63]: /monitors/manage_monitor/
+
+{{% /tab %}}
+{{% tab "Distribution Metrics" %}}
+
+Assign tag keys within [Distribution Metrics][64] (Beta) to create aggregate timeseries by applying sets of tags to a metric, for which a timeseries is created for every combination of tag values within the set.
```

Comment:
Not sure, but this feels like a bit of a run on?

Reply:
Agreed. I have difficulty parsing this sentence.

Reply:
Sadly the sentence is correct, distribution metrics are a quite new concept for DD and tagging with distribution metrics can't really be more simple than that until we get more experience about them.